.github/workflows/test.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           node-version: ${{ matrix.node }}
 
       - run: npm ci --ignore-scripts
-      - run: npm run test --if-present
+      - run: npm run test
```

Comment:
Is this intended?

Reply:
Yes, this repo doesn't use workspaces (afaik) and with `--if-present` we could have false positives after accidentally removing or renaming `"test"` in `package.json`.